ge06_team01/models/stock.py

```diff
@@ -0,0 +1,17 @@
+from odoo import api, models
+
+class StockLot(models.Model):
+    _inherit = 'stock.lot'
+    
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            product_id = vals['product_id']
```

Comment:
You can remove this if you only use it for the search method, you do not need it stored if you only use it once